jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/commands/CollectFiles.java

```diff
@@ -101,16 +101,17 @@ public void execute() throws Exception {
         while(iter.hasNext() && !limiter.isDone()) {
 
             VirtualFile f = iter.next();
-            InputStream stream = limiter.limit(f);
+            try (InputStream stream = limiter.limit(f)) {
 
-            for (Sanitizer sanitizer : this.sanitizers) {
-                if(sanitizer.accepts(f)){
-                    stream = sanitizer.sanitize(stream);
+                InputStream modifiedStream = stream;
+                for (Sanitizer sanitizer : this.sanitizers) {
+                    if (sanitizer.accepts(f)) {
+                        modifiedStream = sanitizer.sanitize(stream);
```

Comment:
Shouldn't this chain, i.e.

modifiedStream = sanitizer.sanitize(modifiedStream);

Reply:
oops